v1/examples/cd_entities/infra-def-entity.yaml

```diff
@@ -0,0 +1,12 @@
+version: 1
+kind: infra-def
+type: KubernetesGcp
```

Comment:
type and variables should go inside spec
@archit-harness can you please confirm?

Reply:
@shalini-agr we will do it in next pr where we will actually write the crud schema.